pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -263,13 +263,32 @@ func (l *LibvirtConnection) GetDomainStats(statsTypes libvirt.DomainStatsTypes,
 			return list, err
 		}
 
+		jobInfo, err := domStat.Domain.GetJobInfo()
+		if err != nil {
+			return list, err
+		}
+
+		jobStatsInfo, err := domStat.Domain.GetJobStats(0)
+		if err != nil {
+			return list, err
+		}
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.DataRemaining)
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.DataProcessed)
+		log.Log.Infof("Debug Job Stats: %d", jobStatsInfo.MemDirtyRate)
+
 		stat := &stats.DomainStats{}
-		err = statsconv.Convert_libvirt_DomainStats_to_stats_DomainStats(statsconv.DomainIdentifier(domStat.Domain), &domStats[i], memStats, stat)
+		//statDomainJob := &stats.DomainJobInfo{}
+
+		err = statsconv.Convert_libvirt_DomainStats_to_stats_DomainStats(statsconv.DomainIdentifier(domStat.Domain), &domStats[i], memStats, jobInfo, stat)
```

Comment:
you probably wanted `jobStatsInfo` here, right?

Reply:
That is correct. Thanks for pointing out. Changed it!